sdk/ml/azure-ai-ml/samples/ml_samples_workspace.py

```diff
@@ -99,9 +102,13 @@ def ml_workspace_config_sample_snippets_entities(self):
         # Example FQDN rule
         pypirule = FqdnDestination(name="pypirule", destination="pypi.org")
 
+        # Example FirewallSku
+        firewallSku = FirewallSku.BASIC
```

Comment:
nit: also can add a comment here to say that FirewallSku is optional and if not provided will default to Standard